pkgs/tools/package-management/nix-doc/default.nix

```
@@ -0,0 +1,29 @@
+{ stdenv, rustPlatform, fetchFromGitHub, boost, nix, pkg-config, ... }:
```

The `...` here isnt required.
```suggestion
{ stdenv, rustPlatform, fetchFromGitHub, boost, nix, pkg-config }:
```

---

Changed.